presto-main/src/main/java/com/facebook/presto/metadata/FunctionManager.java

```diff
@@ -0,0 +1,5 @@
+package com.facebook.presto.metadata;
+
+public class FunctionManager
```

Comment:
Why are you adding this empty class in this commit, and fill it in in the next commit? Just put the entire class in the commit where you have the body.